redaxo/src/core/lib/login/login.php

```diff
@@ -418,15 +418,14 @@ public static function startSession()
             );
 
             $started = rex_timer::measure(__METHOD__, static function () {
+                error_clear_last();
                 return @session_start();
             });
             if (!$started) {
-                $error = error_get_last();
-                if ($error) {
-                    rex_error_handler::handleError($error['type'], $error['message'], $error['file'], $error['line']);
-                } else {
-                    throw new rex_exception('Unable to start session!');
+                if ($error = error_get_last()) {
```

Comment:
Ich denke wir müssen innerhalb des timers den error?by-ref setzen, da er sonst ggf. Hier auch aus der timer implementierung selbst kommen kann?

Reply:
stimmt, könnte passieren, dass ein Fehler aus dem rex_timer hier gecatched wird.

Reply:
Warum ist eigentlich nicht die ganze Logik der Methode in rex_timer drin?
Schließlich wird als Label __METHOD__, also `rex_login::startSession` gesetzt.

Reply:
Session-start() ist oft wg session locks langsam.

Daher hatte ich es in den timer gesetzt. Je mehr code im timer je weniger weiß man welcher davon langsam. Ist/war

Reply:
Ok, ich hab dann jetzt nur die Exceptions noch mit reingenommen.

Reply:
Ah so hast du es gemeint.. passt für mich 👍